compiler-plugin/src/main/kotlin/arrow/meta/phases/analysis/DefaultElementScope.kt

```diff
@@ -432,6 +429,8 @@ class DefaultElementScope(project: Project) : ElementScope {
 
   override val String.annotatedExpression: AnnotatedExpression
     get() = AnnotatedExpression(expression.value as KtAnnotatedExpression)
+  
+  override fun String.file(fileName: String): File = File(delegate.createFile("$fileName.kt", this.trimIndent()))
```

Comment:
Thanks Daniel! I'm going to try to understand the necessary changes:
- I think that `.trimIndent()` is not necessary
- I'd keep the original `filename` and I'd add the extension where it's necessary.